### ISSUE TITLE
[main] Doc issue in file modules/ROOT/pages/storage.adoc

### ISSUE THREAD

**Issue by lindolfomeira** [coreos/fedora-coreos-docs#425]
There seems to be something amiss (at least to me) in some of the examples here. I have the impression that _- size_mib: 0_ is being wrongfully used in place of _- number: 0_ when referencing the partition labels _var, containers_ and _log_ in the examples _Adding a /var partition to the primary disk, Adding a /var/lib/containers partition to the primary disk_ and _Adding /var/log from a secondary disk_, respectively.

**Comment by bgilbert** [comment]
Nope, those are correct.  I agree that they're written in a confusing way, with `size_mib` at the top; we should probably list `label` first.  (Actually, both `number: 0` and `size_mib: 0` are the defaults, so we could drop the field entirely.)
